smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -375,6 +375,15 @@ def devicesList(selector = '') {
         } else if (resp.status == 401) {
             log.warn "Access token is not valid"
             state.lifxAccessToken = null
+        } else if (resp.status == 404 && "$resp.data".startsWith('[error:Could not find location_id') && selector != '') {
```

Comment:
So `resp.data` in this case is a JSON object, right? Meaning that `"$resp.data"` is the same as calling `toString()` on that object? That means that if there are other fields in the response and the order of those fields ever changes the test might break. Wouldn't it be better to say `resp.data?.error == "Could not find location_id"`? It would be more resilient.